documentation/DevelopmentProcess/DevelopmentProcess.tex

```diff
@@ -84,28 +85,39 @@
+\item \textbf{Git}: a version control system for tracking changes and coordinate work among multiple collaborators
 \item \textbf{Master Branch}: the main branch of the GitHub repository
 \item \textbf{PR}: pull request, a request to add changes from one branch into another done via a merge commit or rebase
 \item \textbf{Slack}: a messaging platform for teams to subscribe and publish to channels or send messages between two or more team members
-\item \textbf{DevOps}: development operations
+\item \textbf{Trello}: a web-based project management application utilized by the project team
 \end{itemize}
 
-\section{Team Members}
-The team members are as follows:
+\section{Project Team}
+The project team members consists of:
 \begin{itemize}
-\item Christopher McDonald
-\item Harit Patel
-\item Janak Patel
-\item Jared Rayner
-\item Nisarg Patel
-\item Sam Hamel
-\item Sharon Platkin
+\item Christopher McDonald - Software Engineering and Management
+\item Harit Patel - Software Engineering and Management
+\item Janak Patel - Mechatronics Engineering and Management
+\item Jared Rayner - Mechatronics Engineering and Management
+\item Nisarg Patel - Mechatronics Engineering and Management
+\item Sam Hamel - Software Engineering
+\item Sharon Platkin - Software Engineering
 \end{itemize}
-In addition to the list above, Dr. Alan Wassyng will be the Project Advisor. He will be assisted by Bennett Mackenzie, Nicholas Annable, Stephen Wynn-Williams and Viktor Smirnov for marking and providing advice to the team.
+Additionally, Dr. Alan Wassyng will be the Project Advisor and will advise the project team as needed with challenges and provide guidance. Importantly, the project team will work closely with Nicholas Annable and Viktor Smirnov as well as seek advice from them on frequent basis.
+
+\subsection{Team Organization Overview}
+The core project team has three hierarchical vertical levels: Team Guide, Team Leads and Team Members. At first, the Team Guide is essentially a centralized Project Leader who primarily ensures that the team is executing the project as planned as well as the team is meeting all the project's requirements such as time, budget and specified scope. Moreover, due to the complexity of the project's scope the team has decided to divide the team into two main areas: Software and Hardware. The software team will be primarily responsible for delivering all of the software aspects of the project to ensure that the hardware team is fully supported by software systems to run the hardware. Whereas, the hardware team will be primarily responsible for delivering all of the hardware components of the project such as Electrical, Mechanical and Mechatronics systems. Next, the team has decided to appoint Team Leads for the Software and Hardware team to increase the effectiveness of tasks delegation and ease sub-team management. A visual representation of the team's organizational structure is shown in Figure \ref{fig:teamoverview}.
```

Comment:
Although I don't think this has been explicitly discussed, this can be moved to the roles and responsibilities. I see our team more as a flat organization with some people being SMEs, which does not this section.